dist/gaia-header.js

```diff
@@ -913,11 +918,40 @@ module.exports = component.register('gaia-header', {
   observerStop: function() {
     if (!this.observing) { return; }
     this.observer.disconnect();
+    window.removeEventListener('resize', this);
+    this._resizeHandled = false;
```

Comment:
Can you explain why this is being set several times outside of the rAF below? Looks a bit messy :)